src/components/pages/manage/productions/index/Events.js

```diff
@@ -256,15 +329,33 @@ const Events = ({
               {errorMessage}
             </Alert>
           </Stack>
-        ) : (
+        ) : null}
+
+        {isChangeNameActionVisible ? (
+          <Stack as="div" spacing={3}>
+            <ChangeNameAction
+              onConfirm={onConfirmChangeProductionName}
+              onCancel={onCancelChangeProductionName}
+              productionName={activeProductionName}
+              changedProductionName={changedProductionName}
+              onChangedProductionName={onChangedProductionName}
+            />
+            <Alert visible={!!errorMessage} variant={AlertVariants.DANGER}>
+              {errorMessage}
+            </Alert>
+          </Stack>
+        ) : null}
+
+        {!isAddActionVisible && !isChangeNameActionVisible ? (
           <Actions
             loading={loading}
             activeProductionName={activeProductionName}
             onClickAdd={onClickAdd}
+            onClickChangeName={onClickChangeName}
             onClickDelete={onClickDelete}
             shouldDisableDeleteButton={shouldDisableDeleteButton}
           />
```

Comment:
```suggestion
        ) : !isAddActionVisible ? (
          <Actions
            loading={loading}
            activeProductionName={activeProductionName}
            onClickAdd={onClickAdd}
            onClickChangeName={onClickChangeName}
            onClickDelete={onClickDelete}
            shouldDisableDeleteButton={shouldDisableDeleteButton}
          /> : null}
```

You can also chain multiple conditional renders behind eachother